app/models/tenant.rb

```diff
@@ -33,6 +33,7 @@ class Tenant < ApplicationRecord
   has_many :services, :dependent => :destroy
 
   belongs_to :default_miq_group, :class_name => "MiqGroup", :dependent => :destroy
+  belongs_to :source, :polymorphic => true
```

Comment:
@durandom yes, we was discussing it with @jrafanie and @gtanzillo and we decided to use 'source' - this tenant is created thanks to existence of related EmsCloud or CloudTenant, something like Who is the source which created this tenant ?

or resource is better ? or other ?